examples/adder/adder.py

```diff
@@ -12,15 +12,15 @@ def hello_adder():
     # Create compilation object
     chip = Chip('adder')
 
-    # Specify Design sources
+    # Specify design sources
     chip.input('adder.v')
 
     # Specify pin constraints
     chip.input('adder.pcf')
 
     # Compiler options
     chip.set('option', 'quiet', True)
-    # chip.set('option', 'remote', True)
+    chip.set('option', 'remote', True)
```

Comment:
@aolofsson @sgherbst An important point here:  whatever we put commented into the code will be default behavior for someone who just wants to push go, so do we want that default behavior to be remote or local execution?  My preference is for the default to be local and to provide teaching tools for adding the remote execution option.

Reply:
I noticed this change affected CI tests, so I update adder.py once more to make local default, but added a `-remote` flag so that there is a way to run remotely without editing any code. My guess is that most people would want to try that first since it is the fastest way to get going.

Reply:
remembered about `create_cmdline` so the addition of the `-remote` option is now just one line